IsTheMartin/mrtnmrls
# Image is not positioned correctly in background

## Summary

All images in site such as posts or pages will have a aspect ratio to center image inside div. This also will help to resize for responsiveness.

## Evidence

|Before|After|
|---|---|
|![image](https://github.com/IsTheMartin/mrtnmrls/assets/38388605/71da0000-92c4-4b94-b281-c9b2bf156c99)|![image](https://github.com/IsTheMartin/mrtnmrls/assets/38388605/c6df4e88-8f82-4584-8afc-3d99469caaec)|
|![image](https://github.com/IsTheMartin/mrtnmrls/assets/38388605/bf110f8f-3a18-4a82-be76-ca0d5173fda2)|![image](https://github.com/IsTheMartin/mrtnmrls/assets/38388605/9cbb8707-d258-448d-9ce7-356fc427bf71)|

## Concern Information

To test please follow the next steps:

1. Navigate to `acerca` o `about`
2. Scroll down to seet image
3. Observe image

## Issue ticket

- https://github.com/IsTheMartin/mrtnmrls/issues/17

## Check list

- [ x ] I did add reviewers to this PR
